api/openapi/api.yaml

```diff
@@ -263,7 +263,7 @@ paths:
           name: manifest_record_type
           schema:
             type: string
-            enum: [ "table", "entity", "both"]
+            enum: ["table", "file", "file_w_entities", "combo"]
           description: Manifest storage type.
```

Comment:
Can you add here in the description about what the default is for user? In swagger UI, user could still select an empty field since this is not a required parameter.

Reply:
You might also want to add logic to handle empty selection here in `submit_manifest_route` function in routes.py

Reply:
I can add description of the default (`combo`).

The default is actually handed in `synapse.py` so its fine if a user does not fill out the `manifest_record_type`. This makes the default uniform across API and CLI.

Reply:
I tested with not selecting anything for `manifest_record_type`, and the code would fail: 
```
  File "/Users/lpeng/Documents/schematic-git2/schematic/api/routes.py", line 397, in submit_manifest_route
    manifest_id = metadata_model.submit_metadata_manifest(
  File "/Users/lpeng/Documents/schematic-git2/schematic/schematic/models/metadata.py", line 349, in submit_metadata_manifest
    manifest_id = syn_store.associateMetadataWithFiles(
  File "/Users/lpeng/Documents/schematic-git2/schematic/schematic/store/synapse.py", line 1517, in associateMetadataWithFiles
    return manifest_synapse_file_id
UnboundLocalError: local variable 'manifest_synapse_file_id' referenced before assignment
```

By not selecting anything, I meant something like this: 
<img width="519" alt="Screen Shot 2023-03-10 at 11 20 25 PM" src="https://user-images.githubusercontent.com/55448354/224464526-d37db773-457d-4793-a031-277a0487882c.png">

Reply:
Thanks for catching this, I didn't realize non-responses were handled so differently between the CLI and API. I'll address it!